src/components/AssetTimeseriesPanel/AssetTimeseriesPanel.test.tsx

```diff
@@ -12,24 +13,34 @@ import { TimeseriesPanelPure } from './components/TimeseriesPanelPure';
 
 configure({ adapter: new Adapter() });
 
-sdk.TimeSeries.list = jest.fn();
+const mockedClient: API = {
+  // @ts-ignore
+  timeseries: {
+    list: jest.fn(),
+  },
+};
 
 describe('AssetTimeseriesPanel', () => {
   beforeEach(() => {
     // @ts-ignore
-    sdk.TimeSeries.list.mockResolvedValue({ items: timeseriesList });
+    mockedClient.timeseries.list.mockReturnValue({
+      autoPagingToArray: async () => [timeseriesListV2],
```

Comment:
How does it work? It's supposed to return just array, not array of arrays.
```suggestion
      autoPagingToArray: async () => timeseriesListV2,
```